lib/proxy.js

```diff
@@ -94,9 +94,9 @@ function buildStream (host, options, secure) {
 ////////////////////////////////////////////////////////////
 
 function buildResultingMiddleware (host, options, stream) {
-  console.log(stream)
   return function proxyMiddleware (req, res, next) {
 
+    var _stream = _.clone(stream, true);
```

Comment:
Ah that is my bad, i wasn't understanding what you were asking for in your comment. i thought you wanted me to move the clone before returning the proxyMiddleware.

I am definitely open for changing the styling